Swiftification/Array.swift

```diff
@@ -29,4 +29,35 @@ public extension Array {
         return (indices ~= index ? self[index] : nil)
     }
     
+    /// Index of the first occurrence of item, if found.
+    func indexOf <U: Equatable> (item: U) -> Int? {
+        if item is Element {
+            return self.indexOf({ (object) -> Bool in
+                return (object as! U) == item
+            })
+        }
+        
+        return nil
+    }
+    
+    /// Returns the first occurence of item, if found
+    func find <U: Equatable> (item: U) -> Element? {
```

Comment:
When would you use this instead of `contains`? It seems like if you already have the item, you have the item. `contains` will tell if it is in there.